includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -724,25 +752,32 @@ private function getCommunity() {
 	}
 
 	private function getLocalSitemapUrl() {
-		$default = true; // $wgEnableLocalSitemapPageExt = true; in CommonSettings
-		$localSitemapAvailable = WikiFactory::getVarValueByName(
-			'wgEnableLocalSitemapPageExt', $this->wikiId, false, $default
-		);
-
-		if ( $localSitemapAvailable ) {
-			return $this->getHref( 'local-sitemap' );
+		if ( $this->product !== self::PRODUCT_FANDOMS ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)